<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <!-- This version uses the openai_ros environments -->
    <rosparam command="load" file="$(find MBRL_transport)/config/MBRL_transport_params.yaml" />
    <!-- Launch the training system -->
    <node pkg="MBRL_transport" name="MBRL_transport" type="start_training.py" output="screen" />
</launch>